app/_locales/en/messages.json

```
@@ -3898,6 +3897,21 @@
   "onboardingSrpImport": {
     "message": "Import using Secret Recovery Phrase"
   },
+  "onboardingSrpImportError": {
+    "message": "Not quite right. Double-check your Secret Recovery Phrase and try again."
```

As per DS guidelines, it shouldn't have a full stop in the json
```suggestion
    "message": "Not quite right. Double-check your Secret Recovery Phrase and try again"
```